Client Database System/Client Database System/Product and Services.cpp

```diff
@@ -8,13 +8,74 @@
 
 #include "Product and Services.hpp"
 #include <iostream>
+#include <fstream>
+#include <iomanip>
+#include <string>
 #include <vector>
 
 using namespace std;
 
-void AddPrice(vector<int>& priceVector){
+Products_and_Services::Products_and_Services(){
+    product = "No product";
+    serviceDescription = "No service description";
+}
+
+void Product_and_Services::AddPrice(vector<int>& priceVector){
     int userPrice;
     
     cin >> userPrice;
     priceVector.push_back(userPrice);
 }
+
+void Product_and_Services::ReadFromFile(vector<string>& productVector, vector<string>& descriptionVector){
+}
+
+void Product_and_Services::ListServices(){
+}
+
+void Product_and_Services::AddNewService(string product, string serviceDescription, vector<string>& productVector, vector<string>& descriptionVector){
+    cout << "Input product, service description: ";
+    cin >> product;
+    cin >> serviceDescription;
```

Comment:
Using cin will only read one word. The fix for this would be `getline(cin, serviceDescription);` to get the entire description